### PR TITLE
Transaction

### PR DESCRIPTION
**Pull request by bkm412**
edit smartContractSaga to use getBalance function
create transaction page
removed reactTable in wallet page restored

### CODE REVIEW

File: src/components/local/transaction/PrivateTradesB.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import ReactTable from 'react-table';
+
+export default class PrivateTradesB extends React.Component{
+
+    constructor(props) {
+        super(props);
+
+    }
+
+    state = {
```
Comment (aman-mahalgavan) [inline src/components/local/transaction/PrivateTradesB.js]: the state should be inside the constructor

File: src/containers/walletContainer.js [hunk 2]
```diff
@@ -75,6 +73,7 @@ class WalletContainer extends Component {
 
 
     renderEditable(cellInfo, flag) {
+        console.log(this.props)
```
Comment (aman-mahalgavan) [inline src/containers/walletContainer.js]: please remove all console.logs

File: src/components/local/transaction/PrivateTradesB.js (hunk 1 above)
Comment (bkm412) [reply src/components/local/transaction/PrivateTradesB.js]: I fixed it

File: src/containers/walletContainer.js (hunk 2 above)
Comment (bkm412) [reply src/containers/walletContainer.js]: console.log is removed already at another commit

**Pull request merged**